### PR TITLE
feat: bundle js scripts

### LINKED ISSUE #521: Bundle Fruzhin .js
In the current index.html we have a lot of js script files imported with the script tag. Our ideal case would be if these imports are not supposed to be imported by the end users but are included in the fruzhin.js.

There is example in the answers of this question https://stackoverflow.com/questions/34607252/es6-import-module-from-url how can they be imported in the js.

### PR DESCRIPTION
**Pull request by Grigorov-Georgi**
# Description

- All js scripts have been relocated into a single fruzhin-lib.js file. This file imports all necessary internal and external dependencies. Key objects, such as Blake2b, HTTP, and ED25519, containing functions and variables that users would interact with, are bound to the global 'window' object.

- In the index.html file is imported only fruzhin-lib.js. To initiate the application, the fruzhin.main() method is invoked within index.html.

- How were these changes implemented and what do they affect?
1. TeaVM property 'moduleType' is set to JSModuleType.ES2015 for modules to be supported. 
2. All scripts in JSBody annotation in the java code should use window.fruzhin, when an object/function from fruzhin-lib.js is called.

Fixes https://github.com/LimeChain/Fruzhin/issues/521

## Checklist:
- [x] I have read the [contributing guidelines](https://github.com/LimeChain/Fruzhin/blob/dev/CONTRIBUTING.md).
- [x] My PR title matches the [Conventional Commits spec](https://www.conventionalcommits.org/).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -3,72 +3,11 @@
...
-        //
-        // libp.handle('/dot/sync/warp', ({stream}) => {
-        //
-        //     console.log('Received a warp sync request');
-        //     stream.write('Hello from the other side');
-        //     stream.end();
-        // });
-    }
-    main();
+<script type="module" >
```
Comment (ablax) [inline src/main/webapp/index.html:9]:
```suggestion
<script>
```

the moduel type is obsolete here

**Review by ablax — APPROVED** [review]

File: src/main/webapp/js/fruzhin-lib.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import 'https://unpkg.com/it-pb-stream/dist/index.min.js';
+
+import * as Blake2b from './blake2b.js';
+import * as ED25519 from './ed25519.js';
+import * as HTTP from './http.js';
+import * as Fruzhin from './fruzhin.js'
+
+var startLibp2p = async (bootnodes) => {
+    console.log('Starting libp2p node');
+    console.log(bootnodes);
```
Comment (Zurcusa) [inline src/main/webapp/js/fruzhin-lib.js]: These were removed [here](https://github.com/LimeChain/Fruzhin-Web/commit/9670bd3683e047f8aa1ffa85d3fa4788d7d3046b). I guess we don't need the anymore.

File: src/main/webapp/js/fruzhin-lib.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    // libp.addEventListener('peer:disconnect', (evt) => console.log('Disconnected:', evt.detail.toString()))
+
+    // let pbStream = ItProtobufStream.pbStream;
+    //
+    // libp.handle('/dot/sync/warp', ({stream}) => {
+    //
+    //     console.log('Received a warp sync request');
+    //     stream.write('Hello from the other side');
+    //     stream.end();
+    // });
```
Comment (Zurcusa) [inline src/main/webapp/js/fruzhin-lib.js]: These as well.

**Review by Zurcusa — APPROVED** [review]

File: src/main/webapp/index.html (hunk 1 above)
Comment (Grigorov-Georgi) [reply src/main/webapp/index.html:9]: It doesn't find fruzhin.main() if type="module" is missing

File: src/main/webapp/js/fruzhin-lib.js (hunk 2 above)
Comment (Grigorov-Georgi) [reply src/main/webapp/js/fruzhin-lib.js]: Fixed after merging dev into this branch

File: src/main/webapp/js/fruzhin-lib.js (hunk 3 above)
Comment (Grigorov-Georgi) [reply src/main/webapp/js/fruzhin-lib.js]: Fixed after merging dev into this branch

**Pull request merged**